.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,90 @@
+name: Production deployment 2
+on:
+  push:
+    branches:
+      - 03-19-set_up_continous_deployment_for_splash
+permissions:
+  id-token: write   
+  contents: read
+jobs:
+  deploy-splash-frontend:
+    name: Deploy monoweb/splash-2025
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-north-1
+          role-to-assume: arn:aws:iam::891459268445:role/MonowebProdSplashFrontendCIRole
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+          
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 9.15.7
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22.14.0
+          cache: 'pnpm'
+          
+      - name: Get OIDC Token
+        uses: actions/github-script@v6
+        id: get-oidc-token
+        with:
+          script: |
+            const id_token = await core.getIDToken()
+            core.setOutput('oidc_token', id_token)
+      - uses: dopplerhq/cli-action@v2
+      # Identity comes from the configured doppler account identity 
+      # (see last step https://docs.doppler.com/docs/service-account-identities#configure-the-identity)
+      - run: |
+          doppler oidc login \
+            --api-host https://api.doppler.com \
+            --scope=. \
+            --identity=8f25d2a1-28b9-4f2a-8fb3-115b10f456b2 \
+            --token=${{ steps.get-oidc-token.outputs.oidc_token }}
+      - name: Install dependencies
+        working-directory: apps/splash-2025
```

Comment:
Isn't thiss a part of the monorepo? In that case it should just be from the root